### PR TITLE
Allows greys to be HoP  again.

### PR DESCRIPTION
**Pull request by carshalash**
Essentially back when the poll was originally made the only other option that was considered to still have head roles was greys, while I don't really think that they should have HoS still due to the fact that greys are kind of the weakest physical race among the space station. Plus you know, the hell that is alien security.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
Plasmamen are allowed to be HoS, HoP, and captain, so there *is* precedent for non-humans playing those roles.

It doesn't make sense to restrict *some* non-humans and not others, does it?

**Comment by carshalash** [comment]
I would like to talk with the chicken and see his opinions on implementing it for greys since they were the other race considered besides humans for it all back when the poll was originally made.

**Comment by PJB3005** [comment]
Dude April's fools is over unless you're like living in Hawaii.

**Comment by carshalash** [comment]
PJB killed the memes, time to make this a serious PR.

**Comment by ghost** [comment]
http://ss13.moe/index.php/poll/46

The only response with less votes is the "I dont care" option. Grays + Humans as the sole heads got less votes than allowing all races to be all heads.

**Comment by yclatious** [comment]
Its amazing how we agree to let plasmemes be a thing under a condition, that condition is completely ignored, plasmemes become a thing and then people imediately start trying to get more aliens into captaincy.

Makes ya think.

**Comment by carshalash** [comment]
Greys were still considered the only other viable option for head roles by whoever made the poll at the time. 

If people come to wanting it to be only grey hops I'd be happy to make that change, that way they can only really be at most acting captains until someone comes to fill in the role.

**Comment by SonixApache** [comment]
plasmemes were a mistake
carsh was a mistake

**Comment by yclatious** [comment]
Why are we giving random shitty aliens that are treated like trash or at most treated like meh crew to be given captaincy status when we know that NT default sucks ass? We let plasmemes under two colaborators saying yes, a bhunch of arguments saying no and the lore reason of plasmemes being actual humans once and being NT employees, they still go down on 3 toolboxes but fuck it, we allowed em, so I want a real good reason for why these grey fags are getting captaincy status or HoP status, because if we go by votes, the thread already pointed out vox being colorblind and medium rp being enforced.

**Comment by ghost** [comment]
The person that made the poll had a bias towards grays, if anything this should mean the entire poll is thrown out not accepted as fact that grays are better than other races.

The fairest way to do it would be run another poll, however this time have other races be options as getting the play heads of staff with humans.

**Comment by carshalash** [comment]
Love you too fam, I was thinking of adding dionea into HoS role under the assumption that they would sprint for the hardsuit like most heads do. But then I remembered that the hardsuits don't really negate the weakness. 

I don't want to make this into a "they got cool toys so everyone else should get them too" Kind of thing, I personally don't really like the idea of plasmamen being captain or really greys for that matter. I will probably end up making it HoP only for the greys depending on how feedback for this PR goes. I was trying to think of something for the vox but in the end its kinda harsh but, I think having trader kind of makes up for their inability to play head roles.

**Comment by carshalash** [comment]
> The person that made the poll had a bias towards grays, if anything this should mean the entire poll is thrown out not accepted as fact that grays are better than other races.

I'm pretty sure the reasoning was due to the other races, the others were purple skeletons, skeletons, gypsy birds and normal skeletons. Plasmamen and skeles can be explained as former employees that were injured in another stations plasmafire/genetics incident. Greys can be explained that they're a fairly intelligent race and would be fine with managing things with the HoP role. 

And like I said, i'm 100% willing to change it to just HoP if people here agree on it.

**Comment by Riekland** [comment]
would you ever fuck off now
What actual arguement can you give for nonhumans being heads? You keep getting asked WHY and you never fucking answer

**Comment by ghost** [comment]
>I'm pretty sure the reasoning was due to the other races, the others were purple skeletons, skeletons, gypsy birds and normal skeletons.

And grays are a fucking meme and the most generic alien in history.

**Comment by Kammerjunk** [comment]
Wow we slipped down this slope in less than one day
Impressive

**Comment by yclatious** [comment]
>I personally don't really like the idea of plasmamen being captain or really greys for that matter.
And yet you made a pr to allow them to be captain, while you didnt like the idea?
>Greys can be explained that they're a fairly intelligent race and would be fine with managing things with the HoP role.
They are wierd shitty meme grey aliens that spout ayy lmao, they are not inteligent or any other bullshit, they are a weak meme race at best
>I was thinking of adding dionea into HoS role under the assumption that they would sprint for the hardsuit like most heads do. But then I remembered that the hardsuits don't really negate the weakness.
No offense carsh, but do you expect me to take you seriously if you though that 2.5 times burn damage was negated by a sec hardsuit, made to be weaker than other sec shit, for the purpose of sec to not powergame and use it all the time?

Skeletons are memes, plasmemes are atleast less memey memes and have a semi real reason to be taken seriously when its not Bob or Rat King, vox are fucking niggerbirds that are traders and raiders and should be as far away as possible from any sort of command, ayy lmaos are a joke race, humans are the best. I called slippery damn slope since day one on the topic of trying to readd these and would you look at that, in less than a fucking day. Activates your damn almonds, doesn't it?

**Comment by carshalash** [comment]
I mostly left it like that to gauge opinion on the matter of them being captain, seeing as that part has been the biggest issue so far I removed it. 

And I'm not really a fan of playing diona and wasn't aware of how their damage multiplier worked while armored.

**Comment by yclatious** [comment]
If you wanted to know the opinions of people, open a damn issue discussing it, don't make a pr to make the change, and suposedly, a change you didn't like?

Honestly, I don't see a single reason for a ayy lmao grey to be a HoP, I can smell forced roundstart NT default and shittery from a damn mile away. No.

**Comment by SonixApache** [comment]
>plasmemes are atleast less memey memes and have a semi real reason to be taken seriously

I still don't understand where this comes from.
They're purple
flammable
skeletons
in hazmat suits.

Generic BONES WALKING BONES make more sense than them.

**Comment by JustSumBody** [comment]
Not a single gameplay argument you're making can't also be applied to plasmeme heads(which were a mistake to begin with outside of seeing some of those fancy plasmaman suits back)

Grays are THE scifi alien, our relationship with them dates back to when they helped the egyptian kings build the pyramids/harass Roswell, fits in a lot better on a space station than walking bones or bird people I'd think but that's just like my opinion and such.

Far as MUH POLLS go, none of them should be heads unless another poll is done but my opinion's worth jack shit I suppose

On that note: Plasmaman PR being merged so quickly alongside the 10 or so April Fools PRs with little input rubs me the wrong way somewhat, hate to be that guy but sounds very "SETTLED IT IN THE DISCORD" to me

**Comment by yclatious** [comment]
Sonix, they are essentially walking burnt as fuck corpses in hazmat suits that ignite with oxygen, I say they are less memey because atleast they have internal organs, a attempt at a backstory, a interesting mechanic to them and so on compared to the actual, purely meme race, skeletons, that unless Im mistaken can even play their ribs like a xylophone.

Getting them the ability to be heads was a mistake, I agree Body, but I accepted it on the damn terms of Sawmemes being perma captain banned. That didn't happen so Im back on shitting the plasmemes, they do fit a lot better than the other races, but they are still mostly a joke race, not nearly as much as skeletons, but still not to be taken seriously.

**Comment by Riekland** [comment]
Fuck OFF
You can't use the lore as an excuse here you shithead, no-one fucking reads or plays a character that's true to it
we have TG plasmamen with their stupid fuckign element names and stupid dickheads with human names I hate it

**Comment by ghost** [comment]
scootaloo captains when

**Comment by SonixApache** [comment]
when the name is changed to something that isn't retarded

>upvoting your own PR

what the fuck man

**Comment by ultraplen** [comment]
slippd

**Comment by PJB3005** [comment]
I'm just gonna close this preemptively.

**Comment by yclatious** [comment]
If nobody really does follow lore, then alright, death to the fucking plasmemes, revert now.

Also, am I really the only guy who takes lore seriously? Thats fucking odd.

**Comment by PJB3005** [comment]
@yclatious 

> We let plasmemes under two colaborators saying yes,

"Approving" a PR is only for the code behind it, not for controversial shit like the actual change.

**Comment by SonixApache** [comment]
@PJB3005 he's referring to the two comments posted in the OP of #14498

**Comment by PJB3005** [comment]
Oh

**Comment by yclatious** [comment]
Yeah, they said "sure, lets have plasmemes" and so we had plasmemes. Not like you can fuck up adding a word in a whitelist PJB, the code is piss easy to do, shame the results from it are fucking garbage.

**Comment by FudgePucker** [comment]
No one takes the lore seriously because most of our lore comes from bay station, and suffers the problem of coming from bay station.

Part of our servers culture comes from bay, and by that I mean us not wanting to be anything like bay. They have round start non valid tarjins, the reason we don't is because the people who play them are cancer, snowflake furfags with a ten page character discription. Bay station is our mirror image, most of the server has nightmares about becoming bay. Why would we follow lore from an entirely different culture?

Also greys don't have lore besides AYYY LMAO XDDDDDDDDDDDDDD.

TL;DR nobody on this server gives a shit about lore.

**Comment by Exxion** [comment]
I'm pretty sure the only lore we have that's from Bay is that of dionae

**Comment by FudgePucker** [comment]
Also there are non lore reasons for not allowing non humans to be command staff, Asimov lawset round start.

If we want to endorse autistic snowflake behavior then we would have NT default by default. But fuck that noise, fuck 10 page character descriptions, if you want to be a filthy non human you should put yourself at risk of being door slammed to death.

Non human heads will rush AI upload, then scream on ahelp when they get what's coming to them

**Comment by clusterfack** [comment]
Plasmamen lore is entirely our own, since we created them from scratch.
